app/js/components/orderEntry/RenderOrderType.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import DrugOrderEntry from '../drugOrderEntry';
+import * as orderTypes from './orderTypes';
+
+const RenderOrderType = (props) => {
+  switch (props.orderTypeID) {
+    case orderTypes.LAB_ORDER.id: {
+      return (<h1>I am a the lab order</h1>);
+    }
+    case orderTypes.DRUG_ORDER.id: {
+      return (
+        <DrugOrderEntry
+          outpatientCareSetting={props.outpatientCareSetting}
+          inpatientCareSetting={props.inpatientCareSetting}
+          location={props.location}
+        />);
+    }
+    default: {
+      return (
+        <DrugOrderEntry
+          outpatientCareSetting={props.outpatientCareSetting}
+          inpatientCareSetting={props.inpatientCareSetting}
+          location={props.location}
+        />);
+    }
+  }
+};
+
+export default RenderOrderType;
```

Comment:
Could you add a line at the end of the file

Reply:
Thanks @shakirandagire 
I will add a new line to the end of that file.